hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-resourcemanager/src/main/java/org/apache/hadoop/yarn/server/resourcemanager/scheduler/capacity/AbstractQueueCapacityCalculator.java

```diff
@@ -84,12 +76,6 @@ public abstract float calculateMaximumResource(ResourceCalculationDriver resourc
    *                                  calculation should be made
    */
   public void calculateResourcePrerequisites(ResourceCalculationDriver resourceCalculationDriver) {
```

Comment:
This method is pretty empty, we should either remove it, or make it abstract.

Reply:
You are right, I have moved the resource normalisation calculation to AbsoluteResourceCapacityCalculator, because it does belong there. The only empty prerequisite method is contained in the PercentageCapacityCalculator now.